src/VirtualClient/VirtualClient.Main/profiles/PERF-MEM-STRESSAPPTEST.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Description": "StressAppTest Workload",
+    "Parameters": {
+        "TimeInSeconds": "60",
+        "UseCpuStressfulMemoryCopy": false
+    },
+    "Actions": [
+        {
+            "Type": "StressAppTestExecutor",
+            "Parameters": {
+                "Scenario": "ApplyStress",
+                "CommandLine": "",
```

Comment:
Do you have couple examples in which the commandline would be? Is it different set of tests? Would parser be able to handle them?

Reply:
StressAppTest supports multiple arguments to control factors like amount of memory being tested (-M), no of threads (-m), etc. The CommandLine that will be provided in profile is appended to add other arguments like time to run and logs file. There is a profile validation in place to handle the cases where they might clash.

Reply:
Resolving the comment for now